dlp/dlp_snippets/inspect.go

```diff
@@ -417,3 +433,37 @@ func inspectBigquery(w io.Writer, client *dlp.Client, project string, minLikelih
 }
 
 // [END dlp_inspect_bigquery]
+
+// Convert the custom dictionary word lists and custom regexes to a list of CustomInfoTypes.
+func buildCustomInfoTypes(customDictionaries []string, customRegexes []string) []*dlppb.CustomInfoType {
```

Comment:
How is this sample going to be displayed? I'm afraid it makes the samples above a bit harder to grok, but could be OK as a single function/sample demonstrating how to do this.

Also, the samples don't include `buildCustomInfoTypes`, so it would be difficult to understand when looking at the docs.

Reply:
Ah, I wasn't thinking about how these would be displayed. I deleted the function and just put the logic inline.